docs/experiment/guides/troubleshooting/variant-jumping.md

```diff
@@ -92,3 +92,15 @@ Another common case is simple overlooked implementation error. For example, we'v
 * Inconsistent capitalization, especially with UUIDs.
     * `15a4f7e9-db4e-4c57-82c7-e57a2995803a`
     * `15A4F7E9-DB4E-4C57-82C7-E57A2995803A`
+
+## Removing users who variant jumped
+
+Generally, when doing analysis you want to be careful when removing data as you may bias your results. It is better to understand why users have variant jumped and if you can fix any implementation bugs, so this does not happen again in any subsequent experiments. If you feel you have a legitimate reason for removing users who variant jumped you can follow these steps:
+
+1. Go to the Monitor tab and click "Open in Analytics"
```

Comment:
Our variant jumping charts are for specific variants, wouldn't it be better to create a chart of all users who have jumped for all variants?

Reply:
How would you do that?